tests/integration/__init__.py

```diff
@@ -38,8 +38,7 @@
 import six
 import shutil
 
-
-from cassandra import OperationTimedOut, ReadTimeout, ReadFailure, WriteTimeout, WriteFailure, AlreadyExists,\
+from cassandra import OperationTimedOut, ReadTimeout, ReadFailure, WriteTimeout, WriteFailure, AlreadyExists, \
```

Comment:
I'm not sure about this whitespace changes, they cloud mess up things when we merge code from upstream